tensorflow/python/ops/nn_fused_batchnorm_test.py

```diff
@@ -404,6 +411,8 @@ def testTrainingShape1(self):
             x_shape, dtype, [1], np.float32, use_gpu=True, data_format='NCHW')
       self._test_training(
           x_shape, dtype, [1], np.float32, use_gpu=False, data_format='NHWC')
+      self._test_training(
+          x_shape, dtype, [1], np.float32, use_gpu=False, data_format='NCHW')
 
   def testTrainingShape2(self):
```

Comment:
Can you add a CPU test to the testTrainingShape3 and testBatchNormGradShape3 functions?